src/partitioned-dbs/index.rst

```diff
@@ -0,0 +1,384 @@
+containing the document:
+
+.. code-block:: bash
+
+    shell> curl http://127.0.0.1:5984/my_new_db/_partition/sensor-260
+    {
+      "db_name": "my_new_db",
+      "doc_count": 1,
+      "doc_del_count": 0,
+      "partition": "sensor-260",
+      "sizes": {
+        "active": 244,
+        "external": 347
+      }
+    }
+
+And we can also list all documents in a partition:
+
+.. code-block:: bash
+
+    shell> curl http://127.0.0.1:5984/my_new_db/_partition/sensor-260/_all_docs
+    {"total_rows": 1, "offset": 0, "rows":[
+        {
+            "id":"sensor-260:sensor-reading-ca33c748-2d2c-4ed1-8abf-1bca4d9d03cf",
+            "key":"sensor-260:sensor-reading-ca33c748-2d2c-4ed1-8abf-1bca4d9d03cf",
+            "value": {"rev": "1-05ed6f7abf84250e213fcb847387f6f5"}
+        }
+    ]}
+
+Note that we can use all of the normal bells and whistles available to
+``_all_docs`` requests. Accessing ``_all_docs`` through the
+``/dbname/_partition/name/_all_docs`` endpoint is mostly a convenience
+so that requests are guaranteed to be scoped to a given partition. Users
+are free to use the normal ``/dbname/_all_docs`` to read documents from
+multiple partitions.
+
+Next, we'll create a design document containing our index for
+getting all readings from a given sensor. The map function is similar to
+our earlier example except we've accounted for the change in the document
+id.
+
+.. code-block:: javascript
+
+    function(doc) {
+        if(doc._id.indexOf(":sensor-reading-") < 0) {
+            return;
+        }
+        for(var r in doc.readings) {
+            emit([doc.sensor_id, r[0]], r[1])
+        }
+    }
+
+We can upload our design document and try out a partitioned
+query:
+
+.. code-block:: bash
+
+    shell> cat ddoc.json
+    {
+        "_id": "_design/sensor-readings",
+        "views": {
+            "by_sensor": {
+                "map": "function(doc) { ... }"
+            }
+        }
+    }
+    shell> curl http://127.0.0.1:5984/my_new_db/_partition/sensor-260/_design/sensor-readings/_view/by_sensor
+    {"total_rows":4,"offset":0,"rows":[
+    {"id":"sensor-260:sensor-reading-ca33c748-2d2c-4ed1-8abf-1bca4d9d03cf","key":["sensor-260","0"],"value":null},
+    {"id":"sensor-260:sensor-reading-ca33c748-2d2c-4ed1-8abf-1bca4d9d03cf","key":["sensor-260","1"],"value":null},
+    {"id":"sensor-260:sensor-reading-ca33c748-2d2c-4ed1-8abf-1bca4d9d03cf","key":["sensor-260","2"],"value":null},
+    {"id":"sensor-260:sensor-reading-ca33c748-2d2c-4ed1-8abf-1bca4d9d03cf","key":["sensor-260","3"],"value":null}
+    ]}
+
+Hooray! Our first partitioned query. For experienced users, that may not
+be the most exciting development, given that the only things that have
+changed are a slight tweak to the document id, and accessing views with
+a slightly different path. However, for anyone who likes performance
+improvements, its actually a big deal. By knowing that the view results
```

Comment:
"it's" because "it is" actually a big deal.

Reply:
Addressed in 347a6ce.